cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -487,7 +487,7 @@ pub struct XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, HandleFee>(
 	PhantomData<(WaivedLocations, HandleFee)>,
 );
 impl<WaivedLocations: Contains<MultiLocation>, FeeHandler: HandleFee> FeeManager
-for XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, FeeHandler>
+	for XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, FeeHandler>
```

Comment:
ditto, is problematic changing whitespace for code we do not own.

Reply:
Same as above.